src/hypervisor.c

```diff
@@ -227,82 +228,12 @@ cc_oci_expand_cmdline (struct cc_oci_config *config,
 
 	bytes = g_strdup_printf ("%lu", (unsigned long int)st.st_size);
 
-	/* XXX: Note that "signal=off" ensures that the key sequence
-	 * CONTROL+c will not cause the VM to exit.
-	 */
-	if (! config->console || ! g_utf8_strlen(config->console, LINE_MAX)) {
-
-		config->use_socket_console = true;
-
-		/* Temporary fix for non-console output, since -chardev stdio is not working as expected
-		 * 
-		 * Check if called from docker. Use -chardev pipe as virtualconsole.
-		 * Create symlinks to docker named pipes in the format qemu expects.
-		 *
-		 * Eventually move to using "stdio,id=charconsole0,signal=off"
-		 */
-		if (! config->oci.process.terminal ) {
-
-			config->console = g_build_path ("/",
-					config->bundle_path,
-					"cc-std", NULL);
-
-			g_debug ("no console device provided , so using pipe: %s", config->console);
-
-			g_autofree gchar *init_stdout = g_build_path ("/",
-							config->bundle_path,
-							"init-stdout", NULL);
+	hypervisor_console = g_build_path ("/", config->state.runtime_path,
+			CC_OCI_CONSOLE_SOCKET, NULL);
 
```

Comment:
@amshinde I can see it , I think that is better to make console optional for state, we used to  save in config->console the device used as backend in qemu  but not I dont think is really needed  anymore, the console will be used just at create time.  I also see that runc does not provide attach command, I wonder if need to be removed from cc-oci-runtime as well .